README.md

```diff
@@ -13,7 +13,7 @@
 Aries is a monorepo built from two modules.
 
 1. aries-core
-2. aries-site
+2. aries-site (For designers and developers. This is they documentation site for the HPE Design System.)
```

Comment:
```suggestion
2. aries-site (For designers and developers. This is the documentation site for the HPE Design System.)
```

Reply:
FYI, this was more of a superficial change to force the branch deploy on Netlify.